cluster/gce/windows/README-GCE-Windows-kube-up.md

```diff
@@ -31,22 +31,23 @@ The most straightforward approach to build those binaries is to run `make
 release`. However, that builds binaries for all supported platforms, and can be
 slow. You can speed up the process by following the instructions below to only
 build the necessary binaries.
-```
-# Fetch the PR: https://github.com/pjh/kubernetes/pull/43
-git remote add pjh https://github.com/pjh/kubernetes
-git fetch pjh pull/43/head
 
-# Get the commit hash and cherry-pick the commit to your current branch
-BUILD_WIN_COMMIT=$(git ls-remote pjh | grep refs/pull/43/head | cut -f 1)
-git cherry-pick $BUILD_WIN_COMMIT
+```
+# Apply https://github.com/pjh/kubernetes/pull/43 to your tree:
+curl \
+  https://patch-diff.githubusercontent.com/raw/pjh/kubernetes/pull/43.patch | \
+  git apply
 
-# Build binaries for both Linux and Windows
+# Build binaries for both Linux and Windows:
 make quick-release
 ```
 
-### 2 Create a Kubernetes cluster
+### 2. Create a Kubernetes cluster
 
 You can create a regular Kubernetes cluster or an end-to-end test cluster.
+End-to-end test clusters support running the Kubernetes e2e tests and enable
+some debugging features such as SSH access on the Windows nodes.
+
```

Comment:
Some more explanation may be good:
  - The master control plane is required to run on a Linux node.
  - Setting at least one or two Linux worker nodes are recommended because many default cluster-addons (e.g., kube-dns) need to run on Linux nodes.

Reply:
Done (a few lines below).